generative/README.md

```diff
@@ -6,16 +6,7 @@ The examples are designed as demonstration to showcase the training process for
 ## Installation
 ```
 pip install lpips
-pip install git+https://github.com/Project-MONAI/GenerativeModels.git@0.2.0
-```
-
-Or install it from source:
-```
-pip install lpips
-git clone https://github.com/Project-MONAI/GenerativeModels.git
-cd GenerativeModels/
-python setup.py install
-cd ..
+pip install monai-generative==0.2.2
```

Comment:
I recommend using the latest version `monai-generative` as `monai` in tutorials, but feel free to use `0.2.2` as well.

Reply:
Hi @mingxin-zheng ,

For stability of the tutorial and easy to maintain, I would suggest to use fixed version of other packages.
Refer to our lightning tutorial and catalyst tutorial:
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d_lightning.ipynb
`!pip install -q pytorch-lightning~=2.0`
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/unet_segmentation_3d_catalyst.ipynb
`!python -c "import catalyst" || pip install -q catalyst==20.07`
And also:
https://github.com/Project-MONAI/tutorials/blob/main/modules/TorchIO_MONAI_PyTorch_Lightning.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/modules/integrate_3rd_party_transforms.ipynb

What do you think?

Thanks.

Reply:
Hi @Nic-Ma 
The current scripts and readme files are not covered by the CICD in the tutorials, so I guess either way works fine.
One day when there is some JupyterLab notebook, it will be nice to use the latest release version in the main branch, like `monai` and `monailabel`, because it will help us expose the issues earlier before the release. Regarding the third-party tools, I agree the version should be fixed because we have zero control on them.